src/Util/Util.js

```diff
@@ -180,6 +180,23 @@ const Util = {
     };
   },
 
+  throttle(func, wait) {
```

Comment:
Is this from `lodash`, or did you built it yourself? If you built it yourself, could we add some tests?
Also, could we add some jsDocs?